pkg/apis/core/v1beta1/types.go

```diff
@@ -377,7 +377,7 @@ type CDIConfigSpec struct {
 	// FilesystemOverhead describes the space reserved for overhead when using Filesystem volumes. A value is between 0 and 1, if not defined it is 0.055 (5.5% overhead)
 	FilesystemOverhead *FilesystemOverhead `json:"filesystemOverhead,omitempty"`
 	// Preallocation controls whether storage for DataVolumes should be allocated in advance.
-	Preallocation bool `json:"preallocation,omitempty"`
+	Preallocation *bool `json:"preallocation,omitempty"`
```

Comment:
Is this change intentional? it seems unrelated.

Reply:
Intentional and somewhat related as I was reviewing the config params.  This property is currently unreferenced, though.  But I assume @tomob has plans for it.

Reply:
What do you mean "unreferenced"? What about that? https://github.com/kubevirt/containerized-data-importer/blob/d31591de94dc2c210b55cf6755fb55fb11397e64/pkg/controller/util.go#L639

Reply:
@tomob I updated the spec, not the status

Reply:
Oh, crap.

What I actually wanted to use there is the spec. It cased the bug I talked about today: https://github.com/kubevirt/containerized-data-importer/pull/1565

Reply:
@tomob so do you want to change to `*bool` in your PR? Or keep here?

Reply:
It should be `*bool` because it is optional.  The status field can remain `bool`

Reply:
Also make sure to run make generate, this changes the generated deep copy